sys-apps/grep/grep-3.7.ebuild

```diff
@@ -45,6 +45,11 @@ src_prepare() {
 		src/egrep.sh || die #523898
 
 	default
+
+	# touch generated files after patching m4, to avoid activating maintainer
```

Comment:
Add a note that this can be dropped when loong patch is merged/gone

Reply:
(Do this for all, as with commit patch note bit)

Reply:
Done.